app/controllers/api/v2/mappings_controller.rb

```diff
@@ -5,6 +5,23 @@ class MappingsController < RestfulController
       def searchable_columns
         []
       end
+
+      def update
+        # hack: set the user temporarily so the model hook can reference it, then set it back
+        temp = resource.user
+        resource.user = current_user
+        update_action
+        respond_with_resource
+        resourse.user = temp
+        update_action
```

Comment:
alternatively, would it be ok to set a value temporarily on the model, because we don't actually need to persist it to the database, on the mapping model itself?